content/en/docs/started/workstation/getting-started-linux.md

```diff
@@ -20,18 +15,21 @@ GCP, AWS and Azure.
 
 ### MicroK8s
 
-[MicroK8s](https://microk8s.io) runs natively on most Linux distributions. 
+[MicroK8s](https://microk8s.io) MicroK8s is a lightweight, fast,
+fully-conformant Kubernetes which runs natively on most Linux distributions.
 
 Follow the installation guide for [Kubeflow with MicroK8s](/docs/started/workstation/getting-started-multipass/) to set up MicroK8s and enable Kubeflow.
 
 ## Linux desktop
 
-### Kubeflow appliance
+### MicroK8s
+
+[MicroK8s](https://microk8s.io) MicroK8s is a lightweight, fast,
+fully-conformant Kubernetes which runs natively on most Linux distributions.
```

Comment:
Hi @evilnick Thanks for helping update the guide!

Is this section a duplicate (it appears to be the same as new content on lines 18-19)?

```
[MicroK8s](https://microk8s.io) MicroK8s is a lightweight, fast,
fully-conformant Kubernetes which runs natively on most Linux distributions.
```

Also, maybe it'd be best to have less "marketing"-like language. @RFMVasconcelos WDYT?

E.g.

```
[MicroK8s](...) is a small and fully-conformant Kubernetes that allows for offline development
and runs natively on most Linux distributions.
```

Reply:
Ah right, you're colleagues @RFMVasconcelos and @evilnick 🙂 👍

Reply:
Again, it was a duplicate yes! And a bit "salesy". I believe it is fixed now :)